schema-loader/README.md

```diff
@@ -80,6 +84,10 @@ Create/Delete Cosmos DB schemas
       --no-scaling       Disable auto-scaling for Cosmos DB
   -p, --password=<key>   Cosmos DB key
   -r, --ru=<ru>          Base resource unit
+      --repair-all       Repair tables : it repairs the table metadata of
```

Comment:
If we can switch sub-commands for each database, then isn't it less confusing to change the description for each?
In short, talking about `When using Cosmos DB ...` in the dynamo mode looks a bit weird.

Reply:
Right, I agree. Thank you for pointing this out. 
Having a different `--repair-all` command description only for the Cosmos DB one meant I had to do a bit of refactoring. It was revised in https://github.com/scalar-labs/scalardb/pull/622/commits/f7992a0f3af7da3577c5177a5dfa40b3acd74ea2
Please have a look when you can.